pkg/features/builder.go

```diff
@@ -71,6 +76,10 @@ func (b *FeatureBuilder) Assess(desc string, fn Func) *FeatureBuilder {
 	return b.WithStep(desc, types.LevelAssess, fn)
 }
 
+func (b *FeatureBuilder) AssessWithDescription(name, description string, fn Func) *FeatureBuilder {
+	return b.WithStepDescription(name, description, types.LevelAssess, fn)
+}
```

Comment:
It would be useful for the whole feature (not only each assessment) to have a description too.